dune-project

```
@@ -7,18 +7,25 @@
 (source
  (github ocurrent/opam-ci-check))
 
-(authors "Author Name")
+(authors
+ "Puneeth Chaganti <punchagan@muse-amuse.in>"
+ "Shon Feder <shon.feder@gmail.com>")
 
-(maintainers "Maintainer Name")
+(maintainers
+ "Puneeth Chaganti <punchagan@muse-amuse.in>"
+ "Shon Feder <shon.feder@gmail.com>")
 
-(license LICENSE)
+; TODO: Add a license field
+;(license LICENSE)
 
-(documentation https://url/to/documentation)
+(documentation
+ https://www.ocurrent.org/opam-ci-check/opam-ci-check/index.html)
 
 (package
  (name opam-ci-check)
- (synopsis "A short synopsis")
- (description "A longer description")
+ (synopsis "Opam Repo CI in your CLI")
+ (description
+  "CLI tool to run different steps of the Opam repository CI pipeline locally.")
```

From the perspective of what this tool is going to be when finished, I think it might be a better fit to be something like:

```suggestion
  "CLI tool that runs the checks used in the Opam repository CI.")
```

WDYT?

---

What do you think about advertising the tool as something that package maintainers can use to run parts of the CI locally (instead of relying entirely on the CI pipeline).  This phrasing you suggest for the description, and the phrasing "CLI driving the opam repo CI" for the summary, both pitch the tool as something that drives the opam repo CI.

---

Good point. What about something like this?

```
"CLI to tool that tests whether packages are publishable on the opam repository.

opam-ci-check is used in the opam repo CI and 
can be used locally and in other CI pipelines
to test the deliverability of packages."

---

I like this! Thanks!